tox.ini

```diff
@@ -2,20 +2,20 @@
 envlist =
     flake8
     isort
-    {base-reqs,optional}-py{310,311,312}-dj{32,42,50}
-    {base-reqs,optional}-py{311,312}-dj{50,main}
+    {base-reqs,dev}-py{310,311,312}-dj{32,42,50}
+    {base-reqs,dev}-py{311,312}-dj{50,main}
 
 skip_missing_interpreters=True
 
 [testenv]
 allowlist_externals = coverage
 deps =
-    base-reqs: -r requirements/base.txt
+    base-reqs: -r requirements/dev.txt
     dj32: django>=3.2,<4.0
     dj42: django>=4.2,<5.0
     dj50: django>=5.0,<5.1
     djmain: https://github.com/django/django/archive/main.tar.gz
-    optional: -r requirements/optional.txt
+    dev: -r requirements/dev.txt
```

Comment:
What the idea of testing base reqs and optional ? Seems like base reqs should not include test ones, and thus it makes impossible to run tests on base-reqs.

cc @chrisclark

Reply:
I agree with that.